src/stores/SearchStore.ts

```diff
@@ -4,28 +4,33 @@ import { useLocalStorage } from '@vueuse/core';
 import { computed } from 'vue';
 
 export const SearchSchema = z.object({
-  searchFilters: z.record(z.string())
+  searchFilters: z.record(z.string()),
+  queryFilter: z.string()
 });
 
 export type SearchType = z.infer<typeof SearchSchema>;
```

Comment:
This zod object is only used as a type (we don't call `.parse` anywhere), I think it should be changed to a interface and remove zod.

Could you change it?